Cargo.toml

```diff
@@ -13,28 +13,34 @@ edition = "2018"
 targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
-composable-node = { path = "node", features = [ "composable", "dali" ]  }
+composable-node = { path = "node", features = ["composable", "dali"] }
 color-eyre = { version = "0.5.11", default-features = false }
 
 [features]
-runtime-benchmarks = [ "composable-node/runtime-benchmarks" ]
-std = [ "composable-node/std" ]
-
-
+runtime-benchmarks = ["composable-node/runtime-benchmarks"]
+std = ["composable-node/std"]
 
 [workspace]
 exclude = ["frame/transaction-fee"]
 members = [
-    "runtime/common",
-    "runtime/primitives",
     "frame/*",
+    "frame/assets/rpc",
+    "frame/assets/runtime-api",
+    "frame/crowdloan-rewards/rpc",
+    "frame/crowdloan-rewards/runtime-api",
```

Comment:
This isn't working if we just set the `*`?

Reply:
Seems like it does, I thought that `frame/*` would only match the first level of nested directories. But it works so I removed them :+1: